babel.config.js

```diff
@@ -1,11 +1,9 @@
 module.exports = {
```

Comment:
We're not using babel anymore in `v4` of PatternFly, so your changes here just affect Jest.

Reply:
ok the new tsconfig settings might be interest. These Jest settings are needed because use of [@observable](https://mobx.js.org/refguide/observable-decorator.html) decorator